### DISCUSSION TITLE
FreeBSD supported?

### DISCUSSION THREAD

**Discussion by mimugmail** [nzymedefense/nzyme#363] [Questions]
Hi guys,

just stumbled upon your project due to a mention in ISC SANS diary :) 
Do you know if the code would run on a FreeBSD system or is it linux-only? 
I didn't see a BSD port out there but I'd like to build one if compatible so we can integrate this project into OPNsense Firewall.

Thx!

**Comment by lennartkoopmann** [comment]
Hi! It should
absolutely work on FreeBSD.

Let me know if you need help! I don't see an "official" BSD port in the future because I don't have the time or expertise to properly maintain it but I'm happy to mention and link to it on the downloads page!

**Reply by mimugmail** [reply]
Ok, great. I'll try to add one when time permits.
Do you have a list of dependencies which is needed and not bundled?

**Reply by lennartkoopmann** [reply]
From the top of my head that would be a Java JDK (OpenJDK works great), postgreSQL and libpcap.

The Ubuntu installation requires the following packages:

```
libpcap0.8 openjdk-11-jre-headless postgresql-12 wireless-tools
```

The `wireless-tools` package is required for `iwconfig` but you can really use any program that lets you tune a WiFi adapter to a specific channel (see `channel_hop_command` configuration)
